<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editprofile</class>
 <widget class="QDialog" name="editprofile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>修改资料</string>
  </property>
  <widget class="QLineEdit" name="login_nameInput">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="login_backButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="login_interestInput">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="login_interestLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>利息</string>
   </property>
  </widget>
  <widget class="QComboBox" name="login_typeCombobox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>一年期</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>三年期</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>五年期</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="login_nameLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="login_addressInput">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="login_okButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QLabel" name="login_typeLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>储种</string>
   </property>
  </widget>
  <widget class="QLabel" name="login_addressLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>地址</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
